config/client.go

```diff
@@ -248,7 +248,7 @@ func GenerateLimitedClientConfig(c *model.Config, diagnosticID string, license *
 	props["AndroidAppDownloadLink"] = *c.NativeAppSettings.AndroidAppDownloadLink
 	props["IosAppDownloadLink"] = *c.NativeAppSettings.IosAppDownloadLink
 
-	props["DiagnosticId"] = diagnosticID
+	props["DiagnosticId"] = telemetryID
```

Comment:
Can we also add an additional field called `telemetryId` with the same value, so we can migrate gradually on the clients side.